openedx_tagging/core/tagging/models.py

```diff
@@ -136,20 +139,64 @@ class Taxonomy(models.Model):
             "Indicates that tags in this taxonomy need not be predefined; authors may enter their own tag values."
         ),
     )
+    system_defined = models.BooleanField(
```

Comment:
@pomegranited We need to make this field non-editable after taxonomy creation. Maybe an `editable=False` can work?

Reply:
Fixed with [321fd46](https://github.com/openedx/openedx-learning/pull/62/commits/321fd464bcdc4b1bc6dd39b1175180c0177b95a4).